.eslintrc.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true,
+        "mocha": true
+    },
+    "extends": ["eslint:recommended","plugin:react/recommended","plugin:import/recommended"],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react","import",
+    ],
+    "rules": {
+        "indent": [
+            "error",
+            "tab"
+        ],
+        
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "double"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ]
+    },
+    "settings": {
+        "react": {
+          "version": "latest"
+        }
+      },
+};
```

Comment:
Add a new line at the end of the file, let's always try to remember this.

Reply:
This has been noted and effected

Reply:
Also best to add a rule to take care of this end of file issue in the `.eslintrc.js` file

Reply:
I've added the rule in the `.eslintrc.js` file